onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -163,6 +195,34 @@ def compute_scale_zp(rmin, rmax, qmin, qmax, symmetric=False):
     return [zero_point, scale]
 
 
+def compute_scale_zp_float8(element_type, std):
+    """Calculate the scale s for a float8 type (E4M3FN).
+    The function assumes the coefficient distribution and the float 8
+    distribution are similar to two gaussian laws.
+
+    :return: zero and scale [z, s]
+
+    More details in notebook `quantization_fp8.ipynb
+    <https://github.com/microsoft/onnxruntime/blob/main/docs/python/notebooks/quantization_fp8.ipynb>`_.
```

Comment:
>https://github.com/microsoft/onnxruntime/blob/main/docs/python/notebooks/quantization_fp8.ipynb [](http://example.com/codeflow?start=5&length=95)

we may move this notebook to onnxruntime example later.

Reply:
ok